graphql-social-media-service/Ballerina.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+org = "integration_samples"
+name = "graphql_social_media"
+version = "0.1.0"
+distribution = "2201.4.0"
```

Comment:
Shall we update to 2201.6.0 and recommit Deps.toml too

Reply:
Updated